client/components/App.vue

```diff
@@ -16,6 +16,185 @@
 	</div>
 </template>
 
+<style>
+@import "../css/icons.css";
+@import "../css/chat-and-search.css";
```

Comment:
I feel like a better distinction can be made between chat-and-search and chat-messages but I don't know it. suggestions welcome